new/LIT7073QeneTamayta.xml

```diff
@@ -0,0 +1,80 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7073QeneTamayta" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">">ተመይጠት፡ ጽዮን፡ ጼዋ፡ አጵሎን፡ ነኪር፤</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Tamayṭat Ṣǝyon ṣewā ʾAṗṗǝlon nakkir… (Qǝne of unknown type)</title>
+                <title xml:lang="en" corresp="#t1">The captives of foreign Apollo turned towards Zion…</title>
+                <editor role="generalEditor" key="AB"/>
```

Comment:
I would propose: "Tamayṭat Ṣǝyon ṣewā ʾAṗṗǝlon nakkir (Qǝne of unknown type) 
Zion, the captive of foreign/alien Apollo, went back/returned ..."

Reply:
Thank you!